.github/workflows/preview.yml

```diff
@@ -2,7 +2,6 @@ name: Preview
 
 on:
   pull_request_target:
```

Comment:
I believe this needs to be `pull_request` when no `types:` are specified. I'm not sure tho.

Reply:
From what I [read on docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request_target) seems like they're same but safer to use secrets on?

Reply:
Ah you're right, it needs `target` as it comments on the PR 👍🏻